### ISSUE TITLE
UX - Splt stacked nodes

### ISSUE THREAD

**Issue by Talie5in** [liamcottle/meshtastic-map#29]
When nodes are stacked on each other, as noted it's hard to click in to see a node beneath to see details, could calculate if they are within 10meters of each other for example?

Can tell there is more then one node here due to having "Heard" enabled on another node and you still have a blue line next to it.

![image](https://github.com/liamcottle/meshtastic-map/assets/7875348/d94bc611-87db-48b5-90a4-eb75b4bcf00a)

An another mapping/visual tool I use - https://lukeprior.github.io/nbn-upgrade-map/?suburb=mount-barker&state=SA&commit=main

When there's more then one location there it displays a number on the dot, and when click spreads them out and can click on the individual marker for details.
![image](https://github.com/liamcottle/meshtastic-map/assets/7875348/b29b449e-6d12-4484-b5bc-2c6cd65d4ba4)
![image](https://github.com/liamcottle/meshtastic-map/assets/7875348/6ea27a0a-fcd6-45a3-a404-b02ce1d6ca4c)

**Comment by Talie5in** [comment]
Just realised you do this in further out zooms - just not when down close.

![image](https://github.com/liamcottle/meshtastic-map/assets/7875348/b6888579-7142-4681-9653-5dbe1f13495c)
![image](https://github.com/liamcottle/meshtastic-map/assets/7875348/8f0067e8-1c8a-435d-b8e2-eaed49cdf3eb)

**Comment by liamcottle** [comment]
Yeah, this is the marker cluster plugin. The best fix is probably to adjust the configured zoom level for it, so it still clusters when zoomed in further.

**Comment by hassos246** [comment]
Except that wouldn't work if the nodes have the same location, witch they often get with the approximated position.
